src/platforms/common/enriching-events/tags.mdx

```diff
@@ -1,18 +1,20 @@
 ---
-title: Add Tags
+title: Customize Tags
```

Comment:
This title makes it seem like you _have_ to customize, but we send tags automatically, and we don't want you to customize a tag we already send. 

Suggest:
Tagging Events
Send Tagged Data
Tagged Data
Event Tags
Event Tagging